var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -219,6 +221,9 @@ class Openmpi(AutotoolsPackage):
     provides('mpi@:3.0', when='@1.7.5:')
     provides('mpi@:3.1', when='@2.0.0:')
 
+    if sys.platform != 'darwin':
```

Comment:
numactl not supported on Mac, but is required. Has been pulling system version in